vars/Debian.yml

```diff
@@ -20,3 +19,14 @@ iptables_persistence_package_names:
 
 # The name of the service providing iptables persistence
 iptables_persistence_service_name: netfilter-persistent
+
+# The packages to install to support the common name verification
+# script
+verify_cn_package_names:
+  - libkrb5-dev
+
+# The pip packages to install to support the common name verification
+# script
+verify_cn_pip_package_names:
+  - python-freeipa
+  - requests-gssapi
```

Comment:
I still question whether requests-gssapi is required for any distribution.  I don't see it imported or used anywhere.

Reply:
This is required to use the `login_kerberos` function of `python-freeipa`.
See: https://python-freeipa.readthedocs.io/en/latest/#python_freeipa.client.Client.login_kerberos

Reply:
Got it!